spec/03-plugins/31-proxy-cache/02-access_spec.lua

```diff
@@ -1346,5 +1361,34 @@ do
         assert.same("Bypass", res.headers["X-Cache-Status"])
       end)
     end)
+
+    it("ignore uri case in cache_key", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/my-route/kong",
+      })
```

Comment:
same as above, plus using a less generic path
```suggestion
        path = "/ignore-case/kong",
        headers = {
          host = "route-20.com",
        },
      })
```